neo/rawio/intanrawio.py

```diff
@@ -167,7 +247,15 @@ def _get_signal_size(self, block_index, seg_index, stream_index):
         signal_channels = self.header["signal_channels"][mask]
         channel_ids = signal_channels["id"]
         channel_id_0 = channel_ids[0]
+        if self.file_format == "header-attached":
+            size = self._raw_data[channel_id_0].size
+        elif self.file_format == 'one-file-per-signal':
+            size = self._raw_data[stream_index][:,0].size
+        else:
+            size = self._raw_data[stream_index][0].size
+
         size = self._raw_data[channel_id_0].size
```

Comment:
I think this line got added in the merge but should not be here, right? line 257

Reply:
Thanks. Fixed it. Happened when I was fixing the merge conflicts. So I think for the #1416 I'll wait until after this is merged.

Reply:
You got to master git, man, don't let git merge dictacte your workflow.

[JK, I can take care of the other one after this if you prefer : ) ]

Reply:
You can host a git workshop for me. :)